tests/micro/zephyr/test_zephyr.py

```diff
@@ -391,8 +389,42 @@ def test_byoc_utvm(platform, west_cmd):
         model=model,
         zephyr_board=zephyr_board,
         west_cmd=west_cmd,
+        build_config=build_config,
     )
 
 
+def _make_large_tensors_sess(model, zephyr_board, west_cmd, shape, build_config):
```

Comment:
the "large" part is just from the shape--maybe name as `_make_add_sess_with_shape`?